sequelize/sync_models.js

```diff
@@ -0,0 +1,8 @@
+const postgresConfig = require('../server/config').postgresConfig
```

Comment:
I know you wanted to have sequelize things isolated, but can we move this file to 'scripts' folder? This is not a library/module that's to be imported.

Reply:
Good point. Come to think of it, I will also add some protection to this to ensure it cannot run with `require()`